python/cuml/linear_model/mbsgd_classifier.pyx

```diff
@@ -26,6 +26,17 @@ class MBSGDClassifier(Base, ClassifierMixin):
     """
     Linear models (linear SVM, logistic regression, or linear regression)
     fitted by minimizing a regularized empirical loss with mini-batch SGD.
+    The MBSGD Classifier implementation is experimental and hence a difference
```

Comment:
I would say "it's experimental" and also "and it uses a different algorithm than sklearn's SGDClassifier" as this is minibatched rather than single sample